cmd/world/root/root_test.go

```diff
@@ -95,3 +98,54 @@ func TestExecuteDoctorCommand(t *testing.T) {
 		assert.Check(t, count > 0, "dependencies %q is not listed in dependencies checking", dep)
 	}
 }
+
+func TestCreateStartStopPurge(t *testing.T) {
+	// Create Cardinal
+	gameDir, err := os.MkdirTemp("", "game-template")
+	assert.NilError(t, err)
+
+	// set tea ouput to variable
+	teaOut := &bytes.Buffer{}
+	createCmd := getCreateCmd(teaOut)
+	createCmd.SetArgs([]string{gameDir})
+
+	err = createCmd.Execute()
+	assert.NilError(t, err)
+
+	// Start cardinal
+	os.Chdir(gameDir)
+	rootCmd.SetArgs([]string{"cardinal", "start", "--build", "--detach"})
+	err = rootCmd.Execute()
+	assert.NilError(t, err)
+
+	// Check cardinal health
+	resp, err := http.Get("http://127.0.0.1:3333/health")
+	assert.NilError(t, err)
+	assert.Equal(t, resp.StatusCode, 200)
+	var healthResponse struct {
+		IsServerRunning   bool
+		IsGameLoopRunning bool
+	}
+	err = json.NewDecoder(resp.Body).Decode(&healthResponse)
+	assert.NilError(t, err)
+	assert.Assert(t, healthResponse.IsServerRunning)
+	assert.Assert(t, healthResponse.IsGameLoopRunning)
+
+	// Stop cardinal
+	rootCmd.SetArgs([]string{"cardinal", "stop"})
+	err = rootCmd.Execute()
+	assert.NilError(t, err)
+
+	// Check cardinal health
+	_, err = http.Get("http://127.0.0.1:3333/health")
+	assert.Error(t, err,
+		"Get \"http://127.0.0.1:3333/health\": dial tcp 127.0.0.1:3333: connect: connection refused")
+
+	// Purge cardinal
+	rootCmd.SetArgs([]string{"cardinal", "purge"})
+	err = rootCmd.Execute()
+	assert.NilError(t, err)
+
+	// Remove dir
+	os.RemoveAll(gameDir)
```

Comment:
nit: This should be moved to just below MkdirTemp:
`defer os.RemoveAll(gameDir)`

Reply:
updated